### PR TITLE
chore: update front page

### PR DESCRIPTION
**Pull request by fzipi**
- update first -> 1st
- move up text to form OWASP CRS / PROJECT

### CODE REVIEW

File: config/_default/languages.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ en:
   copyright: "Copyright &copy; {{< current_year >}} CRS Project. All Rights Reserved."
   title: "CRS Project"
   params:
-    description: "The first line of defense"
+    description: "The 1st line of defense"
```
Comment (dune73) [inline config/_default/languages.yaml:8]: I think we should do the claim in capital as in "The 1st Line of Defense".

File: i18n/en.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-  other: OWASP
+  other: OWASP CRS
 
 hero2:
-  other: CRS Project
+  other: Project
 
 subhero:
-  other: The first line of defense
+  other: The 1st line of defense
```
Comment (dune73) [inline i18n/en.yaml:9]: I think we should do the claim in capital as in "The 1st Line of Defense".

**Review by dune73 — DISMISSED** [review]

File: config/_default/languages.yaml (hunk 1 above)
Comment (fzipi) [reply config/_default/languages.yaml:8]: Awesome. Thanks for using suggestions!

File: i18n/en.yaml (hunk 2 above)
Comment (fzipi) [inline i18n/en.yaml:9]:
```suggestion
  other: The 1st Line Of Defense
```

Comment (fzipi) [inline i18n/en.yaml:9]:
```suggestion
  other: The 1st Line Of Defense
```

File: config/_default/languages.yaml (hunk 1 above)
Comment (fzipi) [inline config/_default/languages.yaml:8]:
```suggestion
    description: "The 1st Line Of Defense"
```

**Review by dune73 — APPROVED** [review]

**Pull request merged**